starter-dev-backend/serverless.yml

```
@@ -13,9 +13,8 @@ custom:
 provider:
   name: aws
   profile: starterdev
-  stage: production
+  stage: dev
```

Change this to:
```suggestion
  stage: ${sls:stage}
  region: ${aws:region}
```

See https://www.serverless.com/framework/docs/providers/aws/guide/variables#referencing-serverless-core-variables and https://www.serverless.com/framework/docs/providers/aws/guide/variables#referencing-aws-specific-variables